docs/api.apib

```diff
@@ -287,12 +287,12 @@ Not found.
 + Request
 
   + Parameters
-    + public_key (string, required) - Public key of NextID Avatar to connect to. Should be secp256k1 curve (for now), 65-bytes or 33-bytes long (uncompressed / compressed) and stringified into hex form (`/^0x[0-9a-f]{65,130}$/`)
```

Comment:
Please add a line of `changelog` on top of this file.

Reply:
Sure, done.